examples/PlotPointwiseGFRE.ipynb

```diff
@@ -0,0 +1,212 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Pointwise GFRE applied on RKHS features\n",
+    "================================================================"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Example for the usage of the `skcosmo.metrics.pointwise_global_reconstruction_error` as the pointwise global feature reconstruction error (pointwise GFRE). We apply the pointwise global feature reconstruction error on the degenerate CH4 manifold dataset containing 3 and 4-body features computed with [librascal](https://github.com/cosmo-epfl/librascal). We will show that using reproducing kernel Hilbert space (RKHS) features can improve the quality of the reconstruction with the downside of being less general."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import matplotlib as mpl\n",
+    "\n",
+    "mpl.rc('font', size=20)\n",
+    "\n",
+    "from skcosmo.datasets import load_degenerate_CH4_manifold\n",
+    "from skcosmo.metrics import (\n",
+    "    global_reconstruction_error,\n",
+    "    pointwise_global_reconstruction_error,\n",
+    ")\n",
+    "from skcosmo.preprocessing import StandardFlexibleScaler\n",
+    "from sklearn.preprocessing._data import KernelCenterer\n",
+    "\n",
+    "# load features\n",
+    "degenerate_manifold = load_degenerate_CH4_manifold()\n",
+    "power_spectrum_features = degenerate_manifold.data.SOAP_power_spectrum\n",
+    "bispectrum_features = degenerate_manifold.data.SOAP_bispectrum"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We compare 3-body features with their mapping to the reproducing kernel Hilbert space (RKHS) projected to the sample space using the nonlinear radial basis function (RBF) kernel\n",
+    "\n",
+    "$$ k^{\\textrm{RBF}}(\\mathbf{x},\\mathbf{x}') = \\exp(-\\gamma \\|\\mathbf{x}-\\mathbf{x}'\\|^2),\\quad \\gamma\\in\\mathbb{R}_+ $$\n",
+    "\n",
+    "The projected RKHS features are computed using the eigendecomposition of the positive-definite kernel matrix $K$\n",
+    "\n",
+    "$$ K = ADA^T = AD^{\\frac12}(AD^{\\frac12})^T = \\Phi\\Phi^T$$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def compute_standardized_rbf_rkhs_features(features, gamma):\n",
+    "    # standardize features\n",
+    "    features = StandardFlexibleScaler().fit_transform(features)\n",
+    "    \n",
+    "    # compute \\|x - x\\|^2\n",
+    "    squared_distance = ( np.sum(features ** 2, axis=1)[:, np.newaxis]\n",
+    "                                  + np.sum(features ** 2, axis=1)[np.newaxis, :] - 2 * features.dot(features.T) )\n",
+    "    \n",
```

Comment:
For readability, I would split these lines a bit more, something like

```py
squared_distance = (
    np.sum(features ** 2, axis=1)[:, np.newaxis]
    + np.sum(features ** 2, axis=1)[np.newaxis, :] 
    - 2 * features.dot(features.T)
)
```

Reply:
changed